mappings/DefaultMarkerDecoration.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS abdg DefaultMarkerDecoration
+	FIELD a label Ljava/lang/CharSequence;
+	FIELD b icon Lakhx;
+	METHOD <init> (Ljava/lang/CharSequence;Lakhx;)V
+		ARG 1 label
```

Comment:
```suggestion
	FIELD a text Ljava/lang/CharSequence;
	FIELD b icon Lakhx;
	METHOD <init> (Ljava/lang/CharSequence;Lakhx;)V
		ARG 1 text
```

"Label" usually refers to the control, not the data. `text` works IMO, not sure if there's a better name

Reply:
I'm pretty sure the string constants in the code referred to it as `label`, which is why I chose that name, but yeah, `text` might be better here